README.md

```diff
@@ -106,25 +181,31 @@ Load the frontend app in your browser at http://localhost:3000.
 
 To run [ESLint](https://eslint.org/) checks execute `npm run lint`
 
-The [chai.js](https://www.chaijs.com/) and [jest](https://jestjs.io/) libraries automate tests that run real database operations using [JWT](https://jwt.io/introduction). The tests use a dedicated database named `test_farm`, distinct from the `pg-litefarm` database that the app normally uses .
+The [chai.js](https://www.chaijs.com/) and [jest](https://jestjs.io/) libraries automate tests that run real database operations using [JWT](https://jwt.io/introduction). The tests use a dedicated database named `test_farm`, distinct from the `pg-litefarm` database that the app normally uses.
 
-1. In a terminal, navigate to the `packages/api` folder. 
+1. In a terminal, navigate to the `packages/api` folder.
 2. Execute `npm run migrate:testing:db` to set up the test database.
 3. Execute `npm test` to launch the tests. Or, to generate test coverage information, run `npm test -- --coverage .` and then see the `coverage/index.html` file.
 
 While the tests do attempt to clean up after themselves, it's a good idea to periodically use `psql` to `DROP` and `CREATE` the `test_farm` database, followed by the migrations from step 2 above.
 
+NOTE: if the test suite fails with errors similar to:
+```
+Cannot find module 'core-js/modules/es.object.define-property.js' from '../shared/validation/sensorCSV.js'
+```
+navigate to the `packages/shared` folder, run `npm install` and then try to switch back to `packages/api` folder and invoke test suite again.
+
 ## webapp
 
 To run [ESLint](https://eslint.org/) checks execute `pnpm lint`
 
-Since this is a mobile web application, webapp should be viewed in a mobile view in the browser.
+Sicne this web application is meant to be used on mobile devices, webapp should be viewed in a mobile view in the browser.
```

Comment:
minor spelling error "Since"

Reply:
Thanks for pointing this one out as well!